index.html

```diff
@@ -38,8 +41,190 @@ <h1>I'm Mucha <br> Glad to see <br> you!</h1>
+                <li class="num"><img src="images/Counter.png" alt=""> 2015</li>
+            </ul>
+
+            <p class="tonic-def">A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups
+                required.</p>
+
+            <ul class="stack">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="js">javaScript</li>
+            </ul>
+
+            <div class="button">
+                <button type="button">See Project</button>
+            </div>
+        </div>
+    </div>
+
+    <!-- card 3 -->
+    <div class="work-container3">
+        <div class="portfolio-img">
+            <img src="images/Snapshoot-Portfolio-3.png" alt="">
+        </div>
+
+        <div class="work-content">
+
+
+            <h1 class="tonic">Tonic</h1>
+
+            <ul class="prop">
+                <li class="can">CANOPY</li>
+                <li class="bac"><img src="images/Counter.png" alt=""> Back End Dev</li>
+                <li class="num"><img src="images/Counter.png" alt=""> 2015</li>
+            </ul>
+
+            <p class="tonic-def">A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups
+                required.</p>
+
+            <ul class="stack">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="js">javaScript</li>
+            </ul>
+
+            <div class="button">
+                <button type="button">See Project</button>
+            </div>
+        </div>
+    </div>
+
+    <!-- card-4 -->
+    <div class="work-container4">
+        <div class="portfolio-img">
+            <img src="images/Snapshoot-Portfolio-4.png" alt="">
+        </div>
+
+        <div class="work-content">
+
+
+            <h1 class="tonic">Multi-Post<br>Stories</h1>
+
+            <ul class="prop">
+                <li class="can">CANOPY</li>
+                <li class="bac"><img src="images/Counter.png" alt=""> Back End Dev</li>
+                <li class="num"><img src="images/Counter.png" alt=""> 2015</li>
+            </ul>
+
+            <p class="tonic-def">A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups
+                required.</p>
+
+            <ul class="stack">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="js">javaScript</li>
+            </ul>
+
+            <div class="button">
+                <button type="button">See Project</button>
+            </div>
+        </div>
     </div>
 
+    <!-- about me  -->
+    <div class="introd">
+        <h1>About<br> Myself</h1>
+        <p>I'm a software developer! I can help you <br> build a product, feature or website Look <br> through some
+            of my work and <br> experience! If you like what you see and <br> have a project you need coded, don't
+            <br> hestiate to contact me
+        </p>
```

Comment:
- [ ] I have noticed that in this section, you tried as much as possible to follow the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=kZ9vagFgdeBfwUuQ-0) 👏 . However, its not yet enough as we need to increase the spacing above the title. This will help us to make the portfolio more professional ☺️ 

Your design 😎 
![about space](https://user-images.githubusercontent.com/99394374/219582482-a7aa36fc-a686-4271-ba85-3a4ba74d42f4.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/219582636-5fe8dc43-2821-4046-944e-d9dfce71ad1c.png)